src/app/pages/OnDemandTvPage/index.jsx

```diff
@@ -173,7 +173,7 @@ const OnDemandTvPage = ({ pageData }) => {
           margins={getGroups(true, true, true, true, false, false)}
         >
           <VisuallyHiddenText as="h1" tabIndex="-1" id="content">
-            {brandTitle}, {formattedTimestamp}
+            {`${brandTitle}, ${formattedTimestamp}`}
```

Comment:
Any way we can comment / unit test / provide some context on this?

It seems likely someone will "clean up" this line in the future - eg, make it look like how it was before without realising they've caused a bug

Reply:
Comments on how to remove comments on how to remove comments on how to remove comments on how to remove comments on how to remove comments on how to remove comments on how to remove comments

Reply:
Or chuck it into a variable?

Reply:
I updated the [test](https://github.com/bbc/simorgh/blob/cee4242130bd5fcc8e717d4e9845b346dbc82010/src/app/pages/OnDemandTvPage/index.test.jsx#L76) to make sure it is equal to a single string

Reply:
But I'll add the comment as well to make it clearer